docs/README.md

```diff
@@ -151,11 +151,16 @@ Content-Length: 23011
 
 **Get records for given filters**<br>
 `GET /records`<br>
-`GET /records?projectId=radar-test&userId=testUser&status=ready&limit=10&lastId=11`
+`GET /records?projectId=radar-test&userId=testUser&sourceType=Mp3Audio&status=READY&limit=5&page=1`
 
 ```json
 {
-  "limit": 10,
+  "page": 
```

Comment:
We can also inline these properties instead of wrapping them into a "page" object. @blootsvoets any preference?